src/js/components/CosmosErrorMessage.js

```diff
@@ -79,11 +73,14 @@ class CosmosErrorMessage extends React.Component {
   }
 
   render() {
+    let {className, onResized, wrapperClass} = this.props;
+
     return (
-      <div className={this.props.wrapperClass}>
+      <div className={wrapperClass}>
         {this.getHeader()}
         <CollapsibleErrorMessage
-          onToggle={this.props.onResized}
+          className={className}
```

Comment:
I had to do this, otherwise the modal was not resizing when the message was expanded. I am open to other ideas. (Ps. I cannot run CI tests locally. I don't have a cypress account).